lets-encrypt/activate-ssl.sh

```diff
@@ -294,7 +231,10 @@ https://github.com/nextcloud/vm
 
 The script will now do some cleanup and revert the settings."
 
-# Cleanup
-apt remove certbot -y
-apt autoremove -y
-clear
+    # Cleanup
+    apt remove certbot -y
+    apt autoremove -y
+    clear
+fi
```

Comment:
In my opinion, the cleanup should happen in the function in lib.sh if it fails. That way we don't repeat the code in all files that use the function.